Makefile

```diff
@@ -48,15 +50,25 @@ serve: go-build
 	$(BIN_DIR)/$(APP_NAME)
 
 db-create:
-	docker create -p 5432:5432 -e POSTGRES_PASSWORD=debug -e POSTGRES_USER=postgres -e POSTGRES_DB=postgres --name scs-status-page-api-db postgres:latest
+	${CONTAINER_RUNTIME} create -p 5432:5432 -e POSTGRES_PASSWORD=debug -e POSTGRES_USER=postgres -e POSTGRES_DB=postgres --name scs-${APP_NAME}-db postgres:latest
 
 db-start:
-	docker start scs-status-page-api-db
+	${CONTAINER_RUNTIME} start scs-${APP_NAME}-db
 
 db-stop:
-	docker stop scs-status-page-api-db
+	${CONTAINER_RUNTIME} stop scs-${APP_NAME}-db
 
 db-remove:
-	docker container rm scs-status-page-api-db
+	${CONTAINER_RUNTIME} container rm scs-${APP_NAME}-db
 
 db-restart: db-stop db-remove db-create db-start
+
+container-build:
+	${CONTAINER_RUNTIME} build -t ${APP_NAME}:latest -t ${APP_NAME}:${HASH} -f Containerfile .
+
+container-push-harbor:
+	${CONTAINER_RUNTIME} tag ${APP_NAME}:${HASH} registry.scs.community/status-page/${APP_NAME}:${HASH}
+	${CONTAINER_RUNTIME} tag ${APP_NAME}:latest registry.scs.community/status-page/${APP_NAME}:latest
+
+	${CONTAINER_RUNTIME} push registry.scs.community/status-page/${APP_NAME}:${HASH}
+	${CONTAINER_RUNTIME} push registry.scs.community/status-page/${APP_NAME}:latest
```

Comment:
Maybe use this target in the zuul pipeline, too?

Of course, that potentially includes establishing push access which should be out of scope for this PR review. If not done quickly, we should create an issue for that.